shub/apps/library/views/helpers.py

```diff
@@ -341,3 +341,17 @@ def get_collection(name, retry=True):
         if retry is True and "/" in name:
             name = name.split("/")[0]
             return get_collection(name, retry=False)
+
+
+def get_collection_by_id(id, retry=True):
```

Comment:
Where is this used? And is retry needed? It's easier in the view to do try/except with getting the collection:
```python
try:
    return Collection.objects.get(id=id)
except Collection.DoesNotExist:
    return None
```

Reply:
`retry` is not needed. It's a copy / paste scrub. I've added this helper for consistency, but I have no objection dropping it and putting the needed snippet into the view.

Reply:
Oops. Then is not needed anymore. I forgot to remote it.